src/components/tasks/card/card.module.scss

```diff
@@ -101,3 +118,111 @@ $black: #252525;
   text-align: left;
   cursor: pointer;
 }
+
+.taskTagLevelWrapper{
+  display: flex;
+  padding-top: .5rem;
+  flex-wrap: wrap;
+}
+.taskTagLevelWrapper.editMode{
+  gap: 0.75rem;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.taskTaglevelContainer{
+  display: flex;
+  gap: 0.25rem
+}
+
+.taskTagLevel{
+  position: relative;
+  background-color: hsla(225, 73%, 57%,0.5);
+  padding: 0.35em;
+  border-radius: 20px;
+  border: 2px solid #4169e1;
```

Comment:
Are we not creating variables for color?

Reply:
yes